packages/create/index.js

```diff
@@ -108,19 +108,49 @@ async function run() {
 
   // Tell user that the process is complete
   const manager = getPackageManager();
-  console.log(kleur.green('\n√ Scaffolding complete. You can now run:'));
+  console.log(
+    kleur.green(
+      '\n√ Scaffolding complete. Run the following commands to get started:',
+    ),
+  );
   if (response.path !== process.cwd()) {
     console.log(
       `  ${kleur.bold('cd')} ${path.relative(process.cwd(), response.path)}`,
     );
   }
+
   const boldManager = kleur.bold(manager);
-  if (manager === 'yarn') {
-    console.log(`  ${boldManager}`);
-    console.log(`  ${boldManager} start`);
+
+  if (response.starter == 'default') {
+    if (manager === 'yarn') {
+      console.log(`  ${boldManager}`);
+      console.log(`  ${boldManager} start`);
+    } else {
+      console.log(`  ${boldManager} install`);
+      console.log(`  ${boldManager} start`);
+    }
   } else {
-    console.log(`  ${boldManager} install`);
-    console.log(`  ${boldManager} start`);
+    if (manager === 'yarn') {
+      console.log(kleur.blue('\n√ Start the NextJS server:'));
+      console.log(`  ${kleur.bold('cd')} next`);
+      console.log(`  ${boldManager}`);
+      console.log(`  ${boldManager} run dev`);
+
+      console.log(kleur.blue('\nIn parallel, serve your revideo project:'));
```

Comment:
Also, I think "in another terminal window" might be easier to understand than "in parallel"